modules/modules.py

```diff
@@ -37,13 +44,35 @@ def unload(self, event):
         event["stdout"].write("Unloaded '%s'" % name)
 
     def reload(self, event):
-        name = event["args_split"][0].lower()
+        name = self.module_name if self.module_name != False else event[
```

Comment:
It'd be much better to just make reload_all call

```self.bot.modules.unload_module(name)
self.bot.modules.load_module(name)```

than to have logic that tells a command hook function to not behave like a command hook function.